README.md

```diff
@@ -164,23 +83,23 @@ d3.json(url).then(data => {
 });
 ```
 
-Another example can be found [here](./application/static/js/buildDataTable.js).
+- You can build on our work by creating your own visualizations using the API. Examples for how to integrate the data retrieved from the API into visualizations including charts and maps can be found in [`static/js`](./static/js).
 
-## Rendering the bushfire map locally
+#### Rendering the bushfire map 
 
-To view the live bushfire map on your own machine, you'll need to create and save a `config.py` file in the root directory within the **application** folder of this repository with a Mapbox API key included in the following format:
+To view the live bushfire map on your own machine, you'll need to create and save a `config.py` file in the root directory within the **application** folder of this repository that includes a Mapbox API key in the following format:
 
 ```py
 API_KEY = "your key here"
 ```
 
 More information on Mapbox API keys can be found on [their website](https://docs.mapbox.com/help/how-mapbox-works/access-tokens/).
```

Comment:
Can we change "their website" to "the Mapbox website"

Reply:
I like this section, but might consider making this section more concise something like:

To view the bushfire map locally:

1. Create a `config.py` file in the root directory within the **application folder** of this repository.
2. Add the following line to the `config.py` file:

`
API_KEY = "your key here"
`

Replace "your key here" with your Mapbox API key. More information on Mapbox API keys can be found....